dbt_project.yml

```diff
@@ -11,7 +11,7 @@ models:
     +schema: qualtrics
     +materialized: table
     intermediate:
-      +materialized: view
+      +materialized: view 
```

Comment:
What was the reasoning to materialize these as views opposed to ephemeral? I am open to views but more generally curious.

Reply:
ah just for testing -- switched to ephemeral!